modules/ppcp-button/resources/js/modules/ContextBootstrap/CartBootstap.js

```diff
@@ -40,10 +47,27 @@ class CartBootstrap {
         });
     }
 
+    handleButtonStatus() {
```

Comment:
Seems same function is used in CheckoutBootstap.js, maybe we can extract it to a helper file. Same for `shouldEnable` below.

Reply:
I did the refactor you suggested, also generalized a bit more and was able to cover all bootstraps with the helper functions. That way the code is a bit cleaner.

Also thought of having an Abstract class, but for now I think would add more confusion than benefit.